src/components/Search/index.tsx

```diff
@@ -49,19 +48,22 @@ const Search: React.FC = () => {
     }
   };
 
-  const handleOnChange = (selected: string, value: string) => {
-    if (selected === SearchBarActions.Search) {
-      setSearch(value);
-      sendEvent({ name: "Applied Search", searchExpression: value });
-    }
+  const handleOnChange = (value: string) => {
+    setSearch(value);
+    sendEvent({ name: "Applied Search", searchExpression: value });
   };
 
+  const [containerRef, setContainerRef] = useState<HTMLDivElement | null>(null);
+
   return (
-    <Container>
+    <Container ref={(el) => setContainerRef(el)}>
+      {hasLogs && containerRef && (
+        <SearchBarGuideCue containerRef={containerRef} />
+      )}
```

Comment:
You can use `useRef` here. Although I would argue the hook should be at the top of the component.  :) 
```ts
  const containerRef = useRef<HTMLDivElement>(null);
  return (
    <Container ref={containerRef}>
      {hasLogs && containerRef.current && (
        <SearchBarGuideCue containerRef={containerRef.current} />
      )}
...
```

Reply:
ty! 🙏